src/algorithms/reco/FarForwardNeutralsReconstructionConfig.h

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025 Sebouh Paul
+#pragma once
+#include <float.h>
+#include <DD4hep/Detector.h>
+
+
+namespace eicrecon {
+
+  struct FarForwardNeutralsReconstructionConfig {
+    /** Correction factors for neutrons in the Hcal */
+    std::vector<double>      n_scale_corr_coeff_hcal={-0.11, -1.5, 0};
+    /** Correction factors for gammas in the Hcal */
+    std::vector<double>      gamma_scale_corr_coeff_hcal={0, -0.13, 0};
+    /** rotation from global to local coordinates */
+    double rot_y=-0.025;
```

Comment:
Like above, I'd suggest a more descriptive name here!
```suggestion
    double global_to_local_rotation=-0.025;
```